src/main/java/mate/academy/service/UserService.java

```diff
@@ -10,11 +10,17 @@ public class UserService {
 
     /**
      * Find user by email. All users are stored in <code>private static final User[] users</code>
+     *
      * @param email - the input parameter
      * @return - user if his email is equal to passed email.
      * Return <code>null</code> if there is no suitable user
      */
     public User findByEmail(String email) {
+        for (User u : users) {
```

Comment:
```suggestion
        for (User user  : users) {
```
Don’t  use abbreviations